Packages/Tracking/Core/Runtime/Scripts/HandPoses/HandPoseDetector.cs

```diff
@@ -0,0 +1,373 @@
+using LeapInternal;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.UIElements;
+
+namespace Leap.Unity
+{
+    public class HandPoseDetector : MonoBehaviour
+    {
+        /// <summary>
+        /// Should the detector try to match this pose with both hands?
+        /// Untick to enable single hand poses. E.g. only allowing left hand to activate pose.
+        /// </summary>
+        [HideInInspector]
+        public bool CheckBothHands = true;
+        /// <summary>
+        /// Which hand should the detector check?
+        /// </summary>
+        [HideInInspector]
+        public Chirality ChiralityToCheck;
+        /// <summary>
+        /// List of poses that the detector will check through. This list can be filled with different poses or variations of the same pose.
+        /// Once one of these poses is found, the class will call the "Pose Detected event"
+        /// </summary>
+        [SerializeField]
+        private List<HandPoseScriptableObject> _posesToDetect;
```

Comment:
I think ideally the default fro this wouldn't be a List, so it is more noticeable in the Inspector